src/utils/deviceInfo/deepDeviceInfo.ts

```diff
@@ -161,6 +165,19 @@ class DetailedDeviceInformationProvider {
         if (!model && !vendor) return '';
         return `${model} ${vendor}`;
     }
+
+    private static getDeviceType(dataOrigin: UAParser): DeviceType {
+        const deviceType = dataOrigin.getDevice().type;
+        if (deviceType) {
```

Comment:
Could remove nested if with early return: it makes the code more readable.
```ts
if(!deviceType) {
   return 'desktop';
}
// rest of the code...
if (deviceType.includes('mobile')) {
   return 'mobile';
}
...
```

Reply:
will do, thanks!